debug.md

```diff
@@ -0,0 +1,39 @@
+# assisted-service
+
+## Debug Setup
+
+To debug the application first set the `DEBUG` environment variable to any nonempty value
+
+```shell
+export DEBUG=true
+```
+
+The default remote debug port is 40000 but its configurable by setting the `DEBUG_PORT` env variable:
+
+```shell
+export DEBUG_PORT=8765
+```
+
+Compile the code with debug information and patch the image:
+
+```shell
+skipper make update-debug-minimal
+```
+
+Compile the code with debug information, patch the image and push to your local k8s:
+
+```shell
+skipper make _update-local-image
```

Comment:
Doesn't look like _update-local-image does any of `Compile the code with debug information, patch the image` ?

Reply:
if you set the DEBUG env variable to true,  it's